go.mod_old

```diff
@@ -0,0 +1,89 @@
+module github.com/k0sproject/k0s
```

Comment:
I think we can remove this file and rely on the git history.

Reply:
oh, I've mistakenly committed some local backup file. will remove